addons/website_slides/tests/test_slide_utils.py

```diff
@@ -212,3 +212,34 @@ def test_video_urls(self):
         })
         self.assertEqual('google_drive', slide.video_source_type)
         self.assertEqual('1qU5nHVNbz_r84P_IS5kDzoCuC1h5ZAZR', slide.google_drive_id)
+
+        # test URL from Vimeo
+        slide = Slide.create({
```

Comment:
Could probably be done in a dedicated unit test (test_video_youtube, test_video_vimeo, ...) .